src/app/_components/CheckBox/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './CheckBox';
```

Comment:
🟠 please prefer using named imports/exports

Reply:
okay, fixed it for real this time